api/types/swarm/container.go

```diff
@@ -21,6 +21,28 @@ type DNSConfig struct {
 	Options []string `json:",omitempty"`
 }
 
+// SELinuxContext contains the SELinux labels of the container.
+type SELinuxContext struct {
+	Disable bool
+
+	User  string
+	Role  string
+	Type  string
+	Level string
+}
+
+// CredentialSpec for managed service account (Windows only)
+type CredentialSpec struct {
+	File     string
+	Registry string
+}
+
+// Privileges defines the security options for the container.
+type Privileges struct {
```

Comment:
t.b.h., still a bit on the fence on naming this `Privileges`, as `SELinux` is not really "privileges", "CredentialSpec" perhaps, but if we add `AppArmor` profiles and `Seccomp` profiles, we're really diverging from this.

If this is the "security profile" perhaps `SecurityOptions`, `Security`, or `SecurityProfile`